packages/manager/modules/dedicated-universe-components/src/user/index.js

```diff
@@ -0,0 +1,28 @@
+import angular from 'angular';
+
+import userController from './user.controller';
+import userService from './user.service';
+import userSessionController from './session/user-session.controller';
+import config from '../config';
+
+const moduleName = 'ducUser';
+
+angular
+  .module(moduleName, [])
+  .constant('Billing.constants', {
+    aapiRootPath: config.aapiRootPath,
+    swsProxyRootPath: config.swsProxyRootPath,
+    paymentMeans: [
+      'bankAccount',
+      'paypal',
+      'creditCard',
+      'deferredPaymentAccount',
+    ],
+    target: config.target,
+  })
+  .controller('UserCtrl', userController)
+  .controller('SessionCtrl', userSessionController)
+  .service('User', userService)
```

Comment:
```suggestion
  .controller('ducUserCtrl', userController)
  .controller('ducSessionCtrl', userSessionController)
  .service('ducUser', userService)
```
the service probably needs to be moved but I'm not sure the controller does